tests/bluetooth/controller/ctrl_chmu/prj.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_ZTEST=y
```

Comment:
Question - is it not possible to make a construct similar to -DOVERLAY_CONFIG  so there can be a common conf-file shared between tests, plus individual overlays?

Reply:
it would be possible yes, it is more or less what we did with the kconfig include-file hack. Now we have the unittest board it seems a bit more logical to have a prj.conf file for each 'project'. This can be discussed, and changed in a separate PR if we want to